delivery_carrier_category/models/delivery_carrier.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class DeliveryCarrier(models.Model):
+
+    _inherit = 'delivery.carrier'
+
+    @api.model
+    def _get_default_category_id(self):
+        return self.env['delivery.carrier.category'].search([], limit=1)
```

Comment:
don't understand this function, that seems to return a random category.

Reply:
As categories are ordered by sequence, by default it will load the first (but indeed, in case of equal sequences, it will take the first by id)